src/Nordea/Components/Card.elm

```diff
@@ -108,6 +111,7 @@ view attrs children (Card config) =
     else
         Html.div baseStyle
             ((config.title |> viewMaybe (\title_ -> header [] [ title [] [ Html.text title_ ] ]))
+                :: (config.htmlTitle |> viewMaybe (\htmlTitle -> htmlTitle))
```

Comment:
for the String title, it currently adds
marginBottom 1.5 rem.

Should we do this for the htmlTitle also?
F.ex. by wrapping the htmlTitle here in `div [css [marginBottom (rem 1.5)] [htmlTitle]` ?
@fredrikloberg